sources/engine/Stride.VirtualReality/Stride.VirtualReality.csproj

```diff
@@ -58,22 +49,10 @@
     </None>
   </ItemGroup>
   <ItemGroup>
-    <None Include="GoogleVR\GoogleVR.cpp" />
     <None Include="OculusOVR\OculusOVR.cpp" />
-    <None Include="Fove\Fove.cpp" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Silk.NET.OpenXR" Version="2.15.0" />
+    <PackageReference Include="Silk.NET.OpenXR" Version="2.17.1" />
```

Comment:
We should update all Silk.NET versions referenced in the solution together (see `Stride.Graphics.csproj`) otherwise Perksey's gonna kill us again

Reply:
<img width="333" alt="image" src="https://github.com/stride3d/stride/assets/1094716/4f3ef96b-4f84-4c7e-b3fb-2711bc53b27c">

Reply:
Done.